alert-common/src/main/java/com/synopsys/integration/alert/common/persistence/model/job/DistributionJobModelBuilder.java

```diff
@@ -53,6 +53,7 @@
     private DistributionJobDetailsModel distributionJobDetails;
 
     public DistributionJobModel build() {
+        // TODO validate required fields are present
```

Comment:
Is this going to be done in this PR? or is there a Jira ticket to track this?

Reply:
It will not be done in this PR because this is meant to have parity with the previous model which had no checks for completeness. I will create a Jira issue.